torch/_refs/__init__.py

```diff
@@ -6631,7 +6633,7 @@ def select_scatter(x: TensorLikeType, src: TensorLikeType, dim: int, index: int)
 permute_copy = _make_copy_from_view(aten.permute)
 t_copy = _make_copy_from_view(aten.t)
 transpose_copy = _make_copy_from_view(aten.transpose)
-unbind_copy = _make_copy_from_view(aten.unbind)
+unbind_copy = _make_copy_from_view(aten.unbind, return_none_on_out_variant=True)
```

Comment:
I thought about getting rid of this flag by making a special cased `@register_decomposition(aten.unbind_copy)` but got scared by all the scaffolding we have in `out_wrapper`: https://github.com/pytorch/pytorch/blob/5dd9652389ed7959a842323e4ce063f553710e47/torch/_prims_common/wrappers.py#L233-L405

Reply:
And indeed they matter... Updated the PR to fix some newly exposed bugs.

Reply:
yeah I think that's fine